torch/testing/_internal/common_device_type.py

```diff
@@ -1094,6 +1094,11 @@ def skipCPUIfNoLapack(fn):
     return skipCPUIf(not torch._C.has_lapack, "PyTorch compiled without Lapack")(fn)
 
 
+# Skips a test on CPU if FFT is not available.
+def skipCPUIfNoFFT(fn):
+    return skipCPUIf(not torch._C.has_spectral, "PyTorch is built without FFT support")(fn)
```

Comment:
You need to use this decorator for the `OpInfo` tests as well:
https://github.com/pytorch/pytorch/blob/bfe03120eebc8b2b7c256d08887a782de9b6177c/torch/testing/_internal/common_methods_invocations.py#L2650

Reply:
Thanks, updated!